packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -0,0 +1,286 @@
+//tslint:disable:no-big-function no-commented-code completed-docs
+import { Component, DebugElement, NO_ERRORS_SCHEMA, ViewChild } from "@angular/core";
+import { ComponentFixture, TestBed, async } from "@angular/core/testing";
+import { MatOptionModule, MatSelectModule } from "@angular/material";
```

Comment:
Can you refine this imports from Angular Material? The reason is because having refined imports will ensure that just the modules we use from Angular Material are bundled and not anything else, whereas having a single import makes out bundle bigger by including stuff from Angular Material we don't even use.